### ISSUE TITLE
Console subs using winston always trackTrace as info

### ISSUE THREAD

**Issue by ferantivero** [microsoft/node-diagnostic-channel#57]
I've configured winston in an app with AppInsights-Nodejs and It is successfully sending telemetry to Azure AppInsights.

But [it will always send such telemetry with Information Severity Level](https://github.com/microsoft/node-diagnostic-channel/blob/53c8ab0563de03ef9b1eabd0e0c0a5ec9be27d6a/src/subs/console-sub/console.sub.ts#L11) even when logging errors.

I'd have expected once the winston logger levels matched with the recently [exposed](https://github.com/microsoft/ApplicationInsights-node.js/blob/3adff3bc92b8efb65cc990c15ec301f5c6a64e58/applicationinsights.ts#L16)   [app insights contracts](https://github.com/microsoft/ApplicationInsights-node.js/blob/3adff3bc92b8efb65cc990c15ec301f5c6a64e58/Declarations/Contracts/Generated/SeverityLevel.ts#L7) by using [custom levels](https://github.com/microsoft/node-diagnostic-channel/blob/44885cb49a597364f76de59e207c74f436ed5c0d/src/diagnostic-channel-publishers/tests/winston.spec.ts#L139), it starts sending the severity level properly. But as linked above and [here](https://github.com/microsoft/node-diagnostic-channel/blob/53c8ab0563de03ef9b1eabd0e0c0a5ec9be27d6a/src/subs/console-sub/console.sub.ts#L11), it doesn't seem to be possible.

Am I missing something here?

**Comment by markwolff** [comment]
Winston will be using the winston publisher instead of the console publisher, but the sev levels should still work. Do you have a sample repro for this?

**Comment by ferantivero** [comment]
thanks for the quick turnaround @markwolff 

based on my understanding the problem I'm facing is not at publishing event time, but how is being handled by the subscriber.

Please let me share a repro:

## Init AppInsights
```ts
let appInsights = require('applicationinsights');
appInsights.setup();
appInsights.start();
```

## Init winton
```ts
import { createLogger, format, transports }  from 'winston';
/* alternatively, comment it back in + levels from createLogger...

let AppInsights = require("applicationinsights");
const appInsightsLevels = {
  levels: {
      crit: AppInsights.Contracts.SeverityLevel.Critical,
      error: AppInsights.Contracts.SeverityLevel.Error,
      warn: AppInsights.Contracts.SeverityLevel.Warning,
      info: AppInsights.Contracts.SeverityLevel.Information,
      verbose: AppInsights.Contracts.SeverityLevel.Verbose,
      debug: 5,
      silly: 6
  }
};
*/

const { colorize, combine, timestamp, errors, printf, splat } = format;
const defaultFormat = combine(
    colorize(),
    timestamp(),
    splat(),
    errors(),
    printf(
       ({
            timestamp,
            level,
            message,
            ...rest
        }) => {
            let restString = JSON.stringify(rest, undefined, 2);
            restString = restString === '{}' ? '' : restString;

            return `[${timestamp}] ${level} - ${message} ${restString}`;
        },
    ),
);

const logger = createLogger({
        level: 'debug',
        // levels: appInsightsLevels.levels,
        transports: [
            new transports.Console({ level: 'debug'})
        ],
        format: defaultFormat
    });
```

## Log an error
```
logger.error('create err');
```

## Console (UTC)

![image](https://user-images.githubusercontent.com/158487/57556033-40992d80-734c-11e9-90f2-761293a3e755.png)

## Azure (GMT-3)
![image](https://user-images.githubusercontent.com/158487/57556067-5ad30b80-734c-11e9-882c-62bfad7e2054.png)

**Comment by markwolff** [comment]
Seems to be because the `level` winston passes through has the color encoding included with it. I will have to switch it to use the Symbol field instead like I do for metadata. I'll add this to the next update.
![image](https://user-images.githubusercontent.com/5515583/57560472-74b82280-733b-11e9-8b6f-1d42fe1b88e3.png)

**Comment by ferantivero** [comment]
thanks for the update @markwolff,

For other users facing the same problem, get rid of the ```colorize()``` format will do the trick to get the proper severity level.

![image](https://user-images.githubusercontent.com/158487/57634610-1463f380-757c-11e9-9210-3896a3681ae2.png)

**Comment by markwolff** [comment]
Closing this issue as it was fixed in this repo. Will be included as a part of 1.4.0 in app insights SDK
